src/components/AppointmentConfigModal.vue

```diff
@@ -0,0 +1,289 @@
+<!--
+  - @copyright Copyright (c) 2021 Richard Steinmetz <richard@steinmetz.cloud>
+  -
+  - @author Richard Steinmetz <richard@steinmetz.cloud>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<Modal
+		size="large"
+		@close="$emit('close')">
+		<!-- Wait for the config to become available before rendering the form. -->
+		<div v-if="editing" class="appointment-config-modal">
+			<h2>{{ title }}</h2>
+
+			<div class="appointment-config-modal__form">
+				<fieldset>
+					<TextInput
+						class="appointment-config-modal__form__row"
+						label="Name"
```

Comment:
needs l10n

Reply:
Definitely! Almost all strings are not localized. I added it to the todo list.